docs/design/assetgeneration.md

```diff
@@ -121,8 +121,9 @@ The following graph shows the relationship between the various assets that the i
 
 ![Image depicting the resource dependency graph](resource_dep.svg)
 
-This graph is generated from the [source](resource_dep.dot) using the following command:
+This graph is generated from the using the following command:
 
 ```sh
-dot -Tsvg resource_dep.dot -o ./resource_dep.svg
+$ bin/openshift-install graph --output-file ./docs/design/resource_dep.dot
+$ dot -Tsvg ./docs/design/resource_dep.dot > ./docs/design/resource_dep.svg
```

Comment:
I think we can drop `resource_dep.dot` from version control and just use:

```sh
bin/openshift-install graph | dot -Tsvg >docs/design/resource_dep.svg
```

Also, with `sh` highlighting, you don't want the `$` prompts (more on `sh` vs. `console` in #27.

Reply:
@wking  done.